Samples/InpaintHTTP/TestWebsite/index.html

```diff
@@ -2,15 +2,177 @@
 <html>
 <head>
 	<title>Inpainting API test</title>
+	<meta http-equiv="content-type" content="text/html;charset=utf-8" />
+	<script class="jsbin" src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"></script>
+	<script class="jsbin" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.0/jquery-ui.min.js"></script>
+	<style>
+		body{
+			width: 720px;
+			margin: auto;
+			background-color: #eee;
+		}
+		#content {
+			//background-color: hotpink;
+			margin: auto;
+			//background-image: url(img/logo_transp.png);
+			background-repeat: no-repeat;
+			background-position: 100% 0;
+			border-radius: 10px;
+			min-height: 500px;
+			padding: 15px;
+		}
+		#footer{
+			text-align: center;
+			position: absolute;
+			text-align: center;
+			bottom: 0;
+			text-decoration: none;
+		}
+		h1{
+			padding-left: 10px;
+		}
+		#preview
+		{
+			width: auto;
+			height: auto;
+			min-height: 500px;
+			border-color: gray;
+			border-style: solid;
+			border-radius: 15px;
+			border-width: 3px;
+			margin-bottom: 5px;
+			//background-size: 500px 650px;
```

Comment:
set background-size to `contain` so that an image will fill the container 

```suggestion
			background-size: contain;
```